client/web/src/tree/FileDecorator.scss

```
@@ -6,6 +6,10 @@
         &--active {
             color: $white;
         }
+
+        .theme-redesign &--active {
+            color: var(--text-muted);
+        }
```

```suggestion
        .theme-redesign & {
            &--active {
                color: var(--text-muted);
            }
        }
```

I think? Otherwise it'll expect `theme-redesign` as a child of the above selector

---

Actually I think this is correct, ignore me!